deployment/scripts/operator.sh

```diff
@@ -15,7 +13,7 @@ relay_SIGTERM() {
 
 trap relay_SIGTERM SIGTERM
 
-/operator/initialize-external-operator-identity.sh
+/deployment/initialize-external-operator-identity.sh
```

Comment:
maybe easier to use a variable for `/deployment`  similar to initialize-external-operator-identity.sh

Reply:
I don't think that we need a variable -- this set has to match between Dockerfile and script. Since this script can now be run from the webhook, it made sense to not use the "/operator" path.

Reply:
Added a `DEPLOYMENT_DIR` variable since it was used in two places in 938f839 .